src/main/kotlin/racing/view/RacingUI.kt

```diff
@@ -1,16 +1,19 @@
-package racing
+package racing.view
+
+import racing.domain.RacingCars
+import racing.domain.Winners
 
 object RacingUI {
 
-    fun drawCars(cars: List<Car>) {
-        cars.forEach { car ->
+    fun drawCars(racingCars: RacingCars) {
+        racingCars.cars.forEach { car ->
             println("${car.name} : ${"-".repeat(car.position + 1)}")
         }
         println()
     }
 
-    fun drawWinners(winners: List<Car>) {
-        val winnerMessage = winners
+    fun drawWinners(winners: Winners) {
+        val winnerMessage = winners.cars
```

Comment:
```suggestion
        val winnerMessage = winners.names
```
`RacingUI` 입장에서 `Winner` 객체가 `List<Car>`를 가지고 있다는 정보를 몰라도 된다고 생각합니다.